admin/users/serializers.py

```diff
@@ -18,7 +18,9 @@ def serialize_user(user):
         'osf_link': user.absolute_url,
         'system_tags': user.system_tags,
         'unclaimed': bool(user.unclaimed_records),
-        'requested_deactivation': bool(user.requested_deactivation)
+        'requested_deactivation': bool(user.requested_deactivation),
+        'osf_groups': list(user.osf_groups.all())
```

Comment:
Similar to node and group serializer, I recommend returning serialized groups instead of OSFGroup objects.
```
 'groups': [serialize_group(group, user) for group in user.osf_groups]
```

and then below:
```python
def serialize_group(group, user):
    return {
        'name': group.name,
        'id': group._id,
        'role': user.group_role(group)
    }
```